docs/ru/query_language/functions/encoding_functions.md

```diff
@@ -1,5 +1,68 @@
 # Функции кодирования
 
+## char {#char}
+
+Возвращает символьное значение данного целого значения в соответствии с таблицей ASCII. Принимает несколько числовых аргументов.
+
+**Синтаксис**
+
+```sql
+char(number_1, [number_2, ..., number_n]);
+```
+
+**Параметры**
+
+- `number_1, number_2, ..., number_n` — Числовые аргументы, которые интерпретируются как целые числа. Типы: [Int](../../data_types/int_uint.md), [Float](../../data_types/float.md).
+
+**Возвращаемое значение**
+
+- Строка UTF-8 содержащая символы, полученные путем преобразования числовых аргументов.
```

Comment:
Absolutely unclear. This text only exposes a lack of understanding.
Needs rewrite.

Reply:
How right?